### DISCUSSION TITLE
How do I integrate this into an exisiting kedro project?

### DISCUSSION THREAD

**Discussion by Jason-F** [Galileo-Galilei/kedro-mlflow#308] [Q&A] [answered]
All I want to do is to use mlflow within an existing kedro ML project. What's the easiest way to do this? What I mean is an easier method than what is mentioned on the main page.

I can't use this in the project due to a conflicts occuring, hence why I'm asking of an easy way just use mlflow here

**Comment by Galileo-Galilei** [comment, marked as answer]
Hi @Jason-F, there are a lot of features but the most basic way is to use kedro-mlflow for tracking parameters. 

You can use the following code:

```bash 
pip install kedro-mlflow
kedro mlflow init
kedro run
```

That's all you need to start. You can see your runs by opening the mlflow ui with ``kedro mlflow ui``. 

You may have experienced issues in the past few days if your project is created with ``kedro==0.18.1``. Just upgrade to ``kedro-mlflow ==0.10.0`` that I have jsut released on PyPI and you should be fine. 

If you still experience issues, can you show me the error message, and the python / os / kedro / kedro-mlflow versions you are using?
